### PR TITLE
Docs: Basic README, contributor, env.sample

### PR DESCRIPTION
**Pull request by vogelino**
Docs:
- Basic README,
- Contributor,
- `.env.sample`

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1 @@
+PUBLIC_MAPBOX_ACCESS_TOKEN=pk.lbF7DcivgQQbfJ7wvH3pdEFomTummKGcUnUmCkPxKyVpwH0kKwQbahGbSjFdNkbuaTG4oNXxpPUiDb1.hIM7-DRHBORFldwrenig
```
Comment (raphael-arce) [inline .env.sample:1]: is this a legit mapbox token?

File: README.md [hunk 2]
```diff
@@ -40,6 +40,32 @@ All commands are run from the root of the project, from a terminal:
 | `npm run astro ...`       | Run CLI commands like `astro add`, `astro check` |
 | `npm run astro -- --help` | Get help using the Astro CLI                     |
 
+## Tiles
+
+For this project, we used ALKIS Building Plans available [here](https://fbinter.stadt-berlin.de/fb/index.jsp) which 
+we converted to pbf tiles and .
```
Comment (vogelino) [inline README.md:46]: This sentence seems to miss an end?

**Review by raphael-arce — COMMENTED** [review]
just one question otherwise lgtm 👍

File: .env.sample (hunk 1 above)
Comment (vogelino) [reply .env.sample:1]: Nope. Fake, but I like when sample file look real, this helps compare to the real thing.

**Review by vogelino — COMMENTED** [review]
Great explanation! I just notices a sentence ends with **and**

File: README.md (hunk 2 above)
Comment (raphael-arce) [reply README.md:46]: ah yes, idk what happened here. good catch! will fix in a sec.

**Pull request merged**